miner/tasks.py

```diff
@@ -0,0 +1,86 @@
+import time
+import hashlib
+import math
+from functools import reduce
+from celery import signals
+from transactions.models import Transaction
+from transactions.models import TransactionOutput
+from blocks.models import Block
+
+
+def gen_reward_transaction(val, blk_h):
+    reward_addr = "a5a5da92a90a9ee3b1c640b18be05be07d335127a231ce631b503bfca76876dd3eb871dfaabd5aafe1f749a828577a9d"
```

Comment:
don't hard code it, use some env/settings file to store such values

Reply:
done